cmd/dra-example-kubeletplugin/driver.go

```diff
@@ -28,6 +28,8 @@ import (
 	nasclient "github.com/kubernetes-sigs/dra-example-driver/api/example.com/resource/gpu/nas/v1alpha1/client"
 )
 
+var _ drapbv1.NodeServer = (*driver)(nil)
```

Comment:
```suggestion
var _ drapbv1.NodeServer =&driver{}
```

I think this is a bit more idiomatic.

Reply:
Fixed on both drivers.